### PR TITLE
New Port Strikey Sisters

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Strikey Sisters

## Game Information
- **Title**: Strikey Sisters
- **URL**: https://dyagames.itch.io/strikey-sisters

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Comment by Fraxinus88** [comment]
Kindly review the libgl part of the .sh script. (I don't know what I am doing...)
Also are .png files okay?

File: ports/strikeysisters/Strikey Sisters.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+if [ -f "${controlfolder}/libgl${CFWNAME}.txt" ]; then
```
Comment (kloptops) [inline ports/strikeysisters/Strikey Sisters.sh:16]: This needs to go after line 60 where you cd into `$GAMEDIR` 😄

**Review by kloptops — CHANGES_REQUESTED** [review]
Needs a fix on the libgl_ in your script. 👍

File: ports/strikeysisters/Strikey Sisters.sh (hunk 1 above)
Comment (Fraxinus88) [reply ports/strikeysisters/Strikey Sisters.sh:16]: so get_controls is also after CD gamedir?

Comment (Fraxinus88) [reply ports/strikeysisters/Strikey Sisters.sh]: okies updated (I hope it is right)

**Review by Cebion — APPROVED** [review]

**Pull request merged**